packages/compass-smoke-tests/src/downloads.ts

```diff
@@ -50,5 +58,7 @@ export async function downloadFile({
     fs.createWriteStream(outputPath)
   );
 
+  console.log(outputPath);
```

Comment:
temporary ;)

Reply:
We could introduce use of the `debug` package, that we're using elsewhere?

Reply:
yeah. I suppose that's indicative of just how lazy I am 😆 I'll get to it ASAP..